src/main/java/net/hypercubemc/iris_installer/Installer.java

```diff
@@ -353,12 +356,48 @@ public void start() {
 
         frame.getContentPane().add(topPanel, BorderLayout.NORTH);
         frame.getContentPane().add(bottomPanel, BorderLayout.SOUTH);
+        
+        try {
+            Json versionInfo = Json.read(new URL("https://raw.githubusercontent.com/IrisShaders/Iris-Installer-Maven/master/installer-versions.json"));
```

Comment:
This needs to use the base url variable.

Reply:
Additionally, is there a reason why we are using mJSON instead of the bundled json we depend on in the installer?

Reply:
Fabric installer bundles mjson, however for some reason most of the installer uses the other one (already mentioned it in discord):

https://github.com/FabricMC/fabric-installer/blob/e216bd04ddc3f67818e87c5cf8cc5fe928dc1650/build.gradle#L32
https://discord.com/channels/774352792659820594/774352792659820598/907353295192981554

Didn't see the base url, didn't use it because I was following the one here:
https://github.com/IrisShaders/Iris-Installer/blob/4e64b50aff7fd0bbb8d1764fb46afcedba16413f/src/main/java/net/hypercubemc/iris_installer/Installer.java#L209

Reply:
Actually there's no constant pointing to the base maven url, only to the base files url.
I'll create one though, since it's now used twice.